README.rst

```diff
@@ -1,6 +1,6 @@
 This repository is for homework assignments for `Numerical Software Development
 (NYCU 2022 autumn)
-<https://yyc.solvcon.net/en/latest/nsd/schedule/nsd_22au_nycu.html>`__.
+<https://yyc.solvcon.net/en/latest/nsd/schedule/22au_nycu/schedule.html>`__.
```

Comment:
Could you please avoid including unnecessary changes to your PR?

Reply:
Sorry, I didn't notice it.

Reply:
No problem, but you didn't resolve the issue.  You included an unnecessary change from me cdb97d4 and it should not be there.  Please drop it from the history.

When you fix that you can probably squash the rest of you change into one.